client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -242,13 +273,26 @@ export default function SitesOverview() {
 						<SiteSurveyBanner isDashboardView />
 						<SiteWelcomeBanner isDashboardView />
 						<SiteDowntimeMonitoringUpgradeBanner />
-
 						{ data?.sites && <SiteAddLicenseNotification /> }
 						<SiteContentHeader
-							content={ renderIssueLicenseButton() }
+							content={
+								// render content only on large screens, The buttons for small scren have their own section
+								isLargeScreen &&
+								( selectedLicensesCount > 0 ? (
+									renderIssueLicenseButton()
+								) : (
+									<AddSiteIssueLicenseButtons />
+								) )
+							}
 							pageTitle={ pageTitle }
-							showStickyContent={ !! showIssueLicenseButtonsLargeScreen }
+							// Only renderIssueLicenseButton should be sticky.
+							showStickyContent={ !! ( selectedLicensesCount > 0 && isLargeScreen ) }
```

Comment:
Totally a nitpick: the `!!` here is redundant. It's a shortcut to convert any expression to a Boolean, but `selectedLicensesCount > 0 && isLargeScreen` is already a Boolean.

Reply:
You would think, but typescript freaked out and the tests bombed at the build stage claiming the value could be null instead of a boolean

Reply:
err undefined, not null :)

Reply:
Ahhh, I see. There's a fix for that too, but it's completely out of scope for this PR. 👍